web/gui-v2/src/components/CellStat.jsx

```diff
@@ -25,14 +25,16 @@ const styles = {
   `,
 };
 
+const EMPTY_VALUES = [0, null, '---'];
+
 const CellStat = ({data, colKey}) => {
   return (
     <div css={styles.cell}>
       <div className="val">
         { data?.total === null ? 'n/a' : commas(data.total) }
       </div>
       <div className="rank">
-        { (data?.total === 0 || data?.total === null) ? '---' : (data?.rank && `#${data.rank}`) }
+        { (EMPTY_VALUES.includes(data?.total)) ? '---' : (data?.rank && `#${data.rank}`) }
```

Comment:
As I'm re-reading this, not displaying a rank just because the value is 0 seems potentially problematic for metrics that can go negative. What do you think?